core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/iterable/ChainedIterable.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016-2020 Crown Copyright
+ * Copyright 2022-2026 Crown Copyright
```

Comment:
Something went wrong when updating this!

Reply:
Not sure I understand... The file has been altered so the new version has been updated with the new copyright period...